services/backend/src/core/search/util/getAllProfiles.js

```diff
@@ -293,7 +293,14 @@ async function getAllUsers(language, userId, request) {
 
     if (info.employmentInfo) {
       const employment = info.employmentInfo.translations[0];
-      info.branch = employment ? employment.branch : undefined;
+      info.branch = {};
+      info.branch.name = employment ? employment.branch : undefined;
+      info.branch.acronym = employment
+        ? employment.branch
+            .split(" ")
+            .map((word) => word.charAt(0))
```

Comment:
I think maybe just the following is simpler (since strings are just an array of chars)?

```suggestion
            .map((word) => word[0])
```

Reply:
lol i always thought charAt was better practice but didnt know why. I just learnt its for support for IE7. lol since we dont need to support that and its the backend anyways then ill fix it 😂